pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/github_scraper.go

```diff
@@ -330,8 +259,40 @@ func (ghs *githubScraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 					}
 				}
 			}
+			var prCursor *string
+			var pullRequests []PullRequestNode
+			var perPage int = 100
+
+			prCount, err := getPullRequestCount(ctx, genClient, name, ghs.cfg.GitHubOrg)
+			if err != nil {
+				ghs.logger.Sugar().Errorf("error getting pull request count", zap.Error(err))
+			}
+			ghs.logger.Sugar().Debugf("pull request count: %v for repo %v", prCount, repo)
 
-			ghs.getRepoPullRequestInformation(repoInfo)
+			prPages := getNumPages(float64(perPage), float64(prCount.Repository.PullRequests.TotalCount))
+			ghs.logger.Sugar().Debugf("pull request pages: %v for repo %v", prPages, repo)
```

Comment:
we should add a metric for the number of open pull requests, aka `...open_pull_request.count` and use this data there